operators/vtk_renderer/metadata.json

```diff
@@ -0,0 +1,31 @@
+{
+	"operator": {
+		"name": "vtk_renderer",
+		"authors": [
+			{
+				"name": "Kitware Team",
+				"affiliation": "Kitware Inc"
+			}
+		],
+		"version": "1.0",
+		"changelog": {
+			"1.0": "Initial Release"
+		},
+		"holoscan_sdk": {
+			"minimum_required_version": "0.5.0",
+			"tested_versions": [
+				"0.5.0"
+			]
```

Comment:
It seems like the tested Holoscan SDK version is 1.0.3 based on the Dockerfile accompanying this PR. Can you please confirm?